Content.Shared/Crayon/SharedCrayonComponent.cs

```diff
@@ -21,6 +21,21 @@ public abstract partial class SharedCrayonComponent : Component
         [DataField("color")]
         public Color Color;
 
+        /// <summary>
+        /// Rotation of the resulting decal
+        /// </summary>
+        [DataField("rotation")]
+        public float Rotation;
+
+        [DataField("previewEnabled")]
+        public bool PreviewEnabled;
+
+        [DataField("previewVisible")]
+        public bool PreviewVisible;
+
+        [DataField("opaqueGhost")]
+        public bool OpaqueGhost;
```

Comment:
Names don't need to be specified as long as they match the field name
```suggestion
        [DataField]
        public float Rotation;

        [DataField]
        public bool PreviewEnabled;

        [DataField]
        public bool PreviewVisible;

        [DataField]
        public bool OpaqueGhost;
```